packages/lit-dev-content/src/components/litdev-playground-share-button.ts

```diff
@@ -77,15 +144,36 @@ export class LitDevPlaygroundShareButton extends LitElement {
   @query('litdev-playground-share-long-url')
   private _longUrl?: LitDevPlaygroundShareLongUrl;
 
+  @query('litdev-playground-share-gist')
+  private _gist?: LitDevPlaygroundShareGist;
+
   @query('mwc-snackbar')
   private _snackbar?: Snackbar;
 
+  /**
+   * How the user most recently saved, or undefined if they haven't saved this
+   * pageload.
+   */
+  private _mostRecentSaveType: 'longurl' | 'gist' | undefined = undefined;
```

Comment:
Not sure, kinda just seems like it would be extra indirection, and it's already type-safe as you say. If we wanted to optimize for size, we could define numeric constants. Doesn't seem worth it?

Reply:
Agree!